packages/esm-patient-medications-app/src/order-basket/medication-order-form.component.tsx

```diff
@@ -31,13 +31,15 @@ export interface MedicationOrderFormProps {
   durationUnits: Array<OpenmrsResource>;
   onSign: (finalizedOrder: OrderBasketItem) => void;
   onCancel: () => void;
+  changed: (changed: boolean) => void;
```

Comment:
I'd call this `onChange: () => void;`. The parameter isn't needed since it's always `true` in the code.